producer/network_slice_information_document.go

```diff
@@ -85,7 +85,8 @@ func parseQueryParameter(query url.Values) (plugin.NsselectionQueryParameter, er
 
 // Check if the NF service consumer is authorized
 // TODO: Check if the NF service consumer is legal with local configuration, or possibly after querying NRF through
-//       `nf-id` e.g. Whether the V-NSSF is authorized
+//
+//	`nf-id` e.g. Whether the V-NSSF is authorized
```

Comment:
I think this should be change to a single line like follows.
```go
// TODO: Check if the NF service consumer is legal with local configuration, or possibly after querying NRF through `nf-id` e.g. Whether the V-NSSF is authorized
```
Otherwise, the output of `gofmt` looks strange. What do you think?
```go
// TODO: Check if the NF service consumer is legal with local configuration, or possibly after querying NRF through
//
//	`nf-id` e.g. Whether the V-NSSF is authorized
```

Reply:
Agreed, done.